build/config/common-settings.json

```diff
@@ -257,7 +257,7 @@
    "ums.volume.sdcard1.enabled": true,
    "ums.volume.extsdcard.enabled": true,
    "ums.volume.external.enabled": true,
-   "usb.transfer": "0",
+   "usb.transfer": "auto",
```

Comment:
This will not be populated to the existing profile without a Gecko version bump.